docs/Using Fleet/Supported-host-operating-systems.md

```diff
@@ -6,7 +6,7 @@ Fleet supports the following operating system versions on hosts.
 
 | OS      | Supported version(s)                    |
 | :------ | :-------------------------------------  |
-| MacOS   | 10.12+                                  |
+| MacOS   | 12+                                  |
```

Comment:
```suggestion
| macOS   | 10.12+                                  |
```

This change threw me for a loop.  We're referring to version 10.12 (Monterey) here, because there is no OS12 released.


 
<img width="1100" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/03622d3f-3361-4014-8d7f-25bf1cb2e921">

Reply:
Hi Mike, Monterey is macOS 12 https://setapp.com/how-to/full-list-of-all-macos-versions. 

macOS 10.12 is Sierra. The 10.x versioning is what they did before macOS 11 Big Sur.

Reply:
```suggestion
| MacOS   | 12 (Monterey) and higher                                  |
```

Reply:
ok weird, not sure how I got confused here.  Thanks!

Reply:
@zhumo apparently if you search "macos version market share", like every top result is 2011-era W3schools quality crap:
<img width="1338" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/6bc7ff8f-702c-4918-aacc-26e8eb54ae87">